src/video/SDL_video.c

```diff
@@ -5526,6 +5526,19 @@ void SDL_Vulkan_DestroySurface(VkInstance instance,
     }
 }
 
+SDL_bool SDL_Vulkan_GetPresentationSupport(VkInstance instance,
+                                           VkPhysicalDevice physicalDevice,
+                                           Uint32 queueFamilyIndex)
+{
+    if (_this && instance && physicalDevice) {
+        if (_this->Vulkan_GetPresentationSupport) {
+            return _this->Vulkan_GetPresentationSupport(_this, instance, physicalDevice, queueFamilyIndex);
+        }
+        return SDL_TRUE;
```

Comment:
Is assuming it's always supported when there's no available query valid on all platforms?

Can it be documented, if so?

Reply:
Will write this in now - if the WSI extension doesn't have a query then it's assumed to be always supported.